Runtime/Components/WorldGen.cs

```diff
@@ -198,14 +202,14 @@ private void BuildMesh(int Tx, int Ty, GameObject HoldsMesh)
 		}
 	}
 
-
-    private float getRatio(float pX,float pY,out float height)
+    private float getRatio(float pX, float pY, out float height)
     {
 		height = 0;
-		if (UseCityGrid)
+		if (SpawnPOIs && GetComponent<POI_Gen>() != null)
```

Comment:
Do we want to keep SpawnPOIs, or rely on presence of POI_Gen? Don't feel like we should do both...